eeps/eep-0054.md

```diff
@@ -118,6 +118,12 @@ that was in error, there should be a map element with the argument number
 as the key (that is, `1` for the first argument, `2` for the second, and so on)
 and a `unicode:chardata()` term as the value.
 
+The atoms `general` and `reason` can also be returned in the map.
+`general` represents a generic error that is not attributed for a specific argument
+(for example badarg of io:format("Hello") when the default device is dead).
+`reason` will tell the error pretty printer to print the returned string instead
```

Comment:
Are the values of `reason` and `general` of type `unicode:chardata()` like the other values in the map? It could be clarified.

Reply:
done